src/Client/Updater.php

```diff
@@ -86,20 +76,20 @@ public function validateTarget($targetRepoPath, $targetStream)
         $nextVersion = $version + 1;
         $nextRootContents = $this->getRepoFile("$nextVersion.root.json");
         if ($nextRootContents) {
-          // @todo 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥Add steps do root rotation spec steps 1.3 -> 1.7.
-          //  Not production ready🙀.
+            // @todo 🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥Add steps do root rotation spec steps 1.3 -> 1.7.
+            //  Not production ready🙀.
```

Comment:
All these whitespace fixes aren't really in scope; should we fix them in a separate commit?

Reply:
Also why is phpcs not failing locally for me if these are indented wrong? 🤔

Reply:
Do you disagree with the change?

I rarely object to the exact way a non-controversial change gets done.

Reply:
The change itself is correct, but mixing in out-of-scope CS standards makes review harder. If this were a core issue I would request that the CS fixes be done in a separate patch. https://www.drupal.org/core/scope#creep

I'm also wondering why phpcs isn't failing on HEAD when the indentation is wrong like this. Any ideas?

Reply:
I investigated further, then fixed this in `main` and filed an issue to get PHPCS working for these indentation rules:
https://github.com/php-tuf/php-tuf/issues/30